include/mockturtle/io/write_blif.hpp

```diff
@@ -85,95 +90,139 @@ void write_blif( Ntk const& ntk, std::ostream& os )
   static_assert( has_node_to_index_v<Ntk>, "Ntk does not implement the node_to_index method" );
   static_assert( has_node_function_v<Ntk>, "Ntk does not implement the node_function method" );
 
-  topo_view topo_ntk{ntk};
+  // topo_view topo_ntk{ntk};
```

Comment:
I am not completely sure about this line. Why?

Reply:
I undid this change so networks do not need to be placed in topological order before calling write_blif.